<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formConfdb</class>
 <widget class="QMainWindow" name="formConfdb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>250</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Assitente de Instalação</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>src/config_icon.png</normaloff>src/config_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_title">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>620</width>
      <height>115</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#037f8c;&quot;&gt;BEM VINDO AO ASSISTENTE DE CONFIGURAÇÃO - APOSTADORPRO&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#037f8c;&quot;&gt;GERENCIADOR DE APOSTAS ESPORTIVAS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>200</width>
      <height>45</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>45</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-style: none;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(3, 127, 140);
	border-radius: 12px;
	border-bottom-color: rgb(4, 35, 38);
	border-right-color:  rgb(4, 35, 38);
}

QPushButton:pressed{
	border-style: insetset;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(3, 127, 140);
	border-radius: 8px;
	border-top-color: rgb(4, 35, 38);
	border-left-color:  rgb(4, 35, 38);
}</string>
    </property>
    <property name="text">
     <string>Iniciar Configuração</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_exit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>160</y>
      <width>200</width>
      <height>45</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>45</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-style: none;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(3, 127, 140);
	border-radius: 12px;
	border-bottom-color: rgb(4, 35, 38);
	border-right-color:  rgb(4, 35, 38);
}

QPushButton:pressed{
	border-style: insetset;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(3, 127, 140);
	border-radius: 8px;
	border-top-color: rgb(4, 35, 38);
	border-left-color:  rgb(4, 35, 38);
}</string>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="br_progress">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>580</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>580</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>580</width>
      <height>25</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color: rgb(1, 31, 38);</string>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
